sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -278,4 +286,93 @@ class OrcSourceSuite extends OrcSuite {
       )).get.toString
     }
   }
+
+  test("SPARK-22267 Spark SQL correctly reads parquet files when column order is different") {
```

Comment:
SPARK-22267 is not about Parquet. Why do you add this?

Reply:
Will remove this one, it is only used to help me diagnose the problem.